### DISCUSSION TITLE
[Paddle Backend] Question about adding PaddlePaddle backend

### DISCUSSION THREAD

**Discussion by HydrogenSulfate** [NVIDIA/warp#285] [General]
Hello,

Recently, while using warp, we found it to be a very interesting library. It allows us to accomplish complex tasks in Python that would typically require C++ code. Most importantly, its design supports multiple backends, which greatly facilitates the integration of our own backend code during development. However, due to some legacy issues with our framework, we have currently only adapted the PaddlePaddle backend for warp in my personal repository.

Therefore, if we have prepared the PaddlePaddle backend code for warp along with the corresponding unit tests, would it be possible for us to submit a PR to warp and have it included as one of the backends? Additionally, could we receive some assistance, including code review support?

Thank you

**Comment by mmacklin** [comment]
Hi @HydrogenSulfate, thanks for letting us know - I would be very interested to see your PaddlePaddle integration as we're not too familiar with this framework.

Please submit a PR and we will review.

Cheers,
Miles

**Reply by HydrogenSulfate** [reply]
Thank you very much for your response. We will submit a PR soon after addressing some code issues.
